About/former-board.md

```diff
@@ -0,0 +1,24 @@
+
+
+# Former Board Members:
+
+
+
+## Jono Bacon
+
+![Jono Bacon](website/About/images/jono.jpg)

```

Comment:
For the images, we need absolute paths. Take a look at the CHAOSScon website for an example.

> https://github.com/chaoss/website/blob/master/CHAOSScon/2020EU/speakers.md